src/OpenLoco/TownManager.cpp

```diff
@@ -59,6 +63,58 @@ namespace OpenLoco::TownManager
 
     static auto& rawTowns() { return getGameState().towns; }
 
+    // 0x00497348
+    void sub_497348()
+    {
+        for (auto& town : towns())
+        {
+            town.var_38 = 0;
+            town.population = 0;
+            town.var_34 = 0;
+            memset(town.pad_150, 0, sizeof(town.pad_150));
```

Comment:
rename pad_150 to var_150 now that is is in use. I prefer to use the c++ functions for this so `std::fill(std::begin(town.var_150), std::end(town.var_150), 0);`